### DISCUSSION TITLE
Off-diagonal components of LTC spectra

### DISCUSSION THREAD

**Discussion by apandit2021** [ttadano/alamode#63] [Q&A]
Dear Dr. Tadano,

While calculating the spectrum of the lattice thermal conductivity (LTC) using the ALAMODE (V-1.2.0), I get only the diagonal elements of the LTC spectrum as below. Could you please help me how can I obtain the off-diagonal elements of the LTC spectra as well?
$ more PREFIX.kl_spec
# Temperature [K], Frequency [cm^-1], Thermal Conductivity Spectra (xx, yy, zz) [W/mK * cm]
      0.00      0.00       0.000000       0.000000       0.000000
      0.00      1.00       0.000000       0.000000       0.000000
      0.00      2.00       0.000000       0.000000       0.000000
      0.00      3.00       0.000000       0.000000       0.000000
    .........................
    .........................

The input file/tags I used for this case is:

$ more PREFIX_kl_spec.in
&general
  PREFIX = PREFIX
  MODE = RTA
  FCSXML = abc.xml
  FC2XML = abc_SCPH-2-300K.xml
  NONANALYTIC = 2
  BORNINFO = abc.born
  NKD = 2
  EMIN = 0; EMAX = 1000; DELTA_E = 1
  MASS = * *
/

&cell
60.00 # factor in Bohr unit
0.1000000000    0.0000000000    -0.0000441089
0.0000000000    0.1441677891    0.0000000000
-0.0002100852   0.0000000000    0.6961513514
/

&kpoint
  2  # KPMODE = 2: uniform mesh mode
240 240 1
/

&analysis
KAPPA_SPEC = 1
/

Best,
Abhiyan

**Comment by ttadano** [comment]
Calculation of off-diagonal components of LTC spectra is not supported in the current version of ALAMODE. 
I think the off-diagonal components are almost zero in the current theory. (small non-zero values occur because the group velocity is not symmetrized, I guess.)
